co-authors-plus.php

```diff
@@ -1376,7 +1383,7 @@ public function js_vars() {
 		?>
 			<script type="text/javascript">
 				// AJAX link used for the autosuggest
-				var coAuthorsPlus_ajax_suggest_link = 
+				var coAuthorsPlus_ajax_suggest_link =
```

Comment:
I noticed, that this line was somehow skipped with https://github.com/Automattic/Co-Authors-Plus/pull/771. To avoid that I need to create a separate PR, I'll keep it in this PR, even though it's not related to the problem this PR fixes.